### PR TITLE
build: product-page

### PR DESCRIPTION
**Pull request by Shubhamkr2610**

### CODE REVIEW

File: pages/product-page.html [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+        <aside class="ecomm-sidebar">
+            <div class="sidebar">
+                <h2>Filter</h2>
+                <button>clear</button>
+            </div>
+            <div class="category-filter">
+                <h3>Categories</h3>
+                <div>
+                    <input type="checkbox" class="input-checkbox">
+                    <label for="">mens collection</label>
```
Comment (vishalg8454) [inline pages/product-page.html]: `for` attribute is missing

File: pages/product-page.html [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+                        <li><a href="https://twitter.com/shubhamkr_26" target="_blank" class="social-link"><i
+                                    class="fab fa-twitter-square"></i></a></li>
+                        <li><a href="https://www.instagram.com/i_subhu/" target="_blank" class="social-link"><i
+                                    class="fab fa-instagram"></i></a></li>
+                        <li><a href="https://www.facebook.com/shubham.Lnctu" target="_blank" class="social-link"><i
+                                    class="fab fa-facebook-square"></i></a></li>
+                        <li><a href="https://www.linkedin.com/in/shubham-kumar-6625b8217/" target="_blank"
+                                class="social-link"><i class="fab fa-linkedin"></i></a></li>
+
+
```
Comment (vishalg8454) [inline pages/product-page.html]: take care of white spaces.

File: styles/product-page.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+.badge-dismiss{
+   color: var(--ecomm-wishlist-color);
+}
+.badge-dismiss:hover{
+    color:  var(--ecomm-color-primary);
+}
+.primary-button{   
+    background-color: var(--danger-color);
+    color: var(--button-secondary-color);
+    margin-top: 10px;
```
Comment (vishalg8454) [inline styles/product-page.css:42]: Use rem here.

**Review by vishalg8454 — COMMENTED** [review]
Just minor changes, rest all good.

File: pages/product-page.html [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+                <h1 class="brand-name">Shoe-Shoppers</h1>
+            </a>
+            <div class="nav-search">
+                <input type="text" class="search-input-box" placeholder=" Type to search">
+                <div class="search-button">
+                    <a href="#"><i class="fa fa-search"></i></a>
+                </div>
+            </div>
+            <div class="link-container">
+
```
Comment (GarvG) [inline pages/product-page.html]: removing the white Spaces! Else Everything looks nice to me, good to merge after changes

**Review by GarvG — APPROVED** [review]

**Review by arjun-computer-geek — APPROVED** [review]
looks good to me

**Review by vishalg8454 — APPROVED** [review]

**Pull request merged**